janitor/functions/rename_columns.py

```diff
@@ -8,31 +8,26 @@
 @pf.register_dataframe_method
 @deprecated_alias(old="old_column_name", new="new_column_name")
 def rename_column(
-    df: pd.DataFrame, old_column_name: str, new_column_name: str
+    df: pd.DataFrame,
+    old_column_name: str,
+    new_column_name: str,
 ) -> pd.DataFrame:
     """Rename a column in place.
 
     This method does not mutate the original DataFrame.
 
-    Functional usage syntax:
-
-    ```python
-    df = rename_column(df, "old_column_name", "new_column_name")
-    ```
+    Example: Change the name of column 'a' to 'a_new'.
 
-    Method chaining syntax:
+        >>> import pandas as pd
+        >>> import janitor
+        >>> df = pd.DataFrame({"a": list(range(3)), "b": list("abc")})
+        >>> df.rename_column(old_column_name='a', new_column_name='a_new')
+           a_new  b
+        0      0  a
+        1      1  b
+        2      2  c
 
-    ```python
-    import pandas as pd
-    import janitor
-    df = (
-        pd.DataFrame(...)
-        .rename_column("old_column_name", "new_column_name")
-    )
-    ```
-
-    This is just syntactic sugar/a convenience function
-    for renaming one column at a time.
+    This is just syntactic sugar/a convenience function for renaming one column at a time.
     If you are convinced that there are multiple columns in need of changing,
```

Comment:
Should we remove this line, since rename columns fits in for multiple columns. There is also the larger discussion about dropping the singular forms, since rename columns covers single use as well . Pinging @ericmjl

Reply:
Yes, good point, @samukweku. Let's delete this line.